ace-link-dashboard.el

```diff
@@ -64,18 +67,35 @@
   "Call remove action on item at POINT."
   (dashboard-remove-item-under))
 
-(defun ace-link-dashboard--collect ()
-  "Collect all widgets in the current `dashboard-mode' buffer."
+(defun ace-link-dashboard--on-unicode-symbol-p (point)
+  "Return t if POINT is on a unicode symbol."
+  (eq 'unicode (char-charset (char-after point))))
+
+(defun ace-link-dashboard--in-section-p ()
+  "Return t if point is in a known section of `dashboard-mode' buffer."
+  (ignore-error user-error
+    (not (null (dashboard--current-section)))))
+
+(defun ace-link-dashboard--collect (&optional in-section)
+  "Collect all widgets in the current `dashboard-mode' buffer.
+When IN-SECTION is non-nil, only collect widgets within a known section
+of `dashboard-mode' buffer."
   (save-excursion
     (let ((previous-point (window-start))
           (candidates nil)
           (next-widget-point (lambda ()
-                               (progn (widget-move 1)
-                                      (point)))))
+                               (widget-move 1)
+                               (point))))
       (goto-char (window-start))
       (while (< previous-point (funcall next-widget-point))
         (setq previous-point (point))
-        (push (cons (widget-at previous-point) previous-point) candidates))
+        (when (or (and in-section (ace-link-dashboard--in-section-p))
+                  (not in-section))
+          (push (cons (widget-at previous-point)
```

Comment:
I was thinking in move the `if` to another function.
```emacs-lisp
(defun ace-link-dashboard--widget-point (point)
"Return the POINT where avy overlay will be display.
Avy don't show the correct overlay when point is over an icon"
  (if (eq 'unicode (char-charset (char-after point)))
    (+ 2 point)
    point))
```
And this line changes to:
```emacs-lisp
(push (cons (widget-at previous-point) (ace-link-dashboard--widget-point previous-point)) candidates)
```
Or something like that, to have the `push` as clean as possible.

Reply:
you can push the changes to me branch.

Reply:
Sure.